### DISCUSSION TITLE
Discussion forum has been opened here

### DISCUSSION THREAD

**Discussion by matsduf** [dotse/zonemaster#1295] [Announcements]
All users of Zonemaster are welcome with questions and comments under the "General Discussions" category. Questions could be of any aspect of Zonemaster such as how to use it, what does some message mean, how to install it or how to adapt it for my needs.

**Comment by tgreenx** [comment]
Great initiative ! Looking forward to it.

**Comment by matsduf** [comment]
Discussion completed.
